set.mm

```diff
@@ -461739,6 +461767,24 @@ its graph has a given second element (that is, function value).
       EIKUHUIUOWIWK $.
   $}
 
+  $( Elementhood in a open integer interval, except its lower bound.
+     (Contributed by Thierry Arnoux, 1-Jan-2024.) $)
+  fzne1 $p |- ( ( K e. ( M ... N ) /\ K =/= M ) -> K e. ( ( M + 1 ) ... N ) )
+      $=
+    ( wne cfz co wcel wceq wn c1 caddc df-ne wo cuz cfv elfzuz2 elfzp12 syl ibi
+    wb orcanai sylan2b ) ABDABCEFGZABHZIABJKFCEFGZABLUCUDUEUCUDUEMZUCCBNOGUCUFT
+    ABCPABCQRSUAUB $.
+
+  $( Elementhood in an half-open interval, except the lower bound.
```

Comment:
"in a half-open ..."

Reply:
The comment is somehow misleading, maybe: "Elementhood of an integer and its predecessor in finite intervals of integers."